api/envoy/type/grpc/grpc_status.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package envoy.type.grpc;
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.type.grpc";
+option java_outer_classname = "GrpcStatusProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package = "envoy.type.grpc.v3";
+
+// [#protodoc-title: GRPC status codes]
+
+// GRPC response codes supported.
```

Comment:
I'm really surprised we don't already have this defined somewhere or it's not defined in a Google proto somewhere. @lizan @kyessenov?